src/binding/python/RecordComponent.cpp

```diff
@@ -531,42 +531,54 @@ load_chunk(RecordComponent & r, py::buffer & buffer, Offset const & offset, Exte
         }
     }
 
```

Comment:
I don't really know if this part is necessary. But it looks like we forgot it earlier? Copied this from the other overload of `load_chunk`.
This fix is irrelevant to the use-after-free issue that I saw.